draft-deepspace-ip-architecture.xml

```diff
@@ -216,6 +216,23 @@
+          <li><t>Path MTU Discovery - Internet transports typically include probing algorithms and rely on some combination of ICMP messages  from the network and/or end-to-end acknowledgements in order to infer the maximum packet sizes that can be used at any time without introducing unnecessary IP fragmentation.  Because for interplanetary cases, feedback may be slow or unrelated to the current or future paths by the time it arrives, it may be more beneficial to simply rely on known / pre-arranged path MTU limitations that can be communicated between interplanetary providers and other connected providers and users.  Especially for higher bandwidth links, it may be beneficial for this to be significantly larger than the minimum Internet MTU values that are normally assumed.</t></li>
+          <li><t>Multiple Streams - Several transports allow for applications to send/receive multiple independent streams of application data, rather than requiring separate connections (and handshakes, and state) for each stream.  Due to the need to leverage pre-established state and make efficient use of connectivity opportunities, these capabilities can be valuable to leverage for interplanetary use, however, specific mechanisms may need to be evaluated/tuned/modified e.g. to avoid additional round trip times.</t></li>
+          <li><t>Multipath Transport - There may be a variety of different physical paths between planets (e.g. using relays or direct links), and using multiple paths simultaneously is a common way to increase reliability for mission data.  However, typical Internet multipath transport algorithms may only switch between paths (e.g. for failover), or allow different streams to use different paths, rather than replicating data.  In this case, such replication would need to be done at a higher layer through separate connections per path, if required by a mission.</t></li>
+          <li><t>FEC - Within the transport layer, Forward Error Correction is not a common feature in most Internet use today, though it can be integrated with different stacks.  Because it can allow for data recovery in the case of packet losses, without suffering extra round trips, or requiring bidirectional connectivity, this may be valuable to incorporate/enable in future interplanetary transport stacks.</t></li>
```

Comment:
FEC is increasingly common with advanced optical systems. It is completely buried and invisible to everyone else.

Reply:
Some QUIC stacks have FEC, and of course there are things like FLUTE and FECFRAME over RTP that IETF has standardized for FEC in transport layer.

Maybe we should say that it's generally unexpected to be needed in the transport layer, because space links are generally designed to have sufficient link margin and FEC that there are very few unrepairable frames / frame losses.